pkg/kp/rcstore/store.go

```diff
@@ -46,3 +52,33 @@ type Store interface {
 	// The two output channels will be closed in response.
 	Watch(rc *fields.RC, quit <-chan struct{}) (<-chan struct{}, <-chan error)
 }
+
+func RCPath(rcId fields.ID) (string, error) {
```

Comment:
You know, don't bother with this. What you've got now is safer than what came before.